oop/homework.py

```diff
@@ -78,6 +104,19 @@ class Cheetah:
 
     """
 
+    def eat(self, product):
+        food_energy = {'gazelle': 30, 'rabbit': 15}
```

Comment:
you could declare food_energy dict outside the method.
In that case, you don't have to rewrite parent method.

Reply:
Cool. Thanks.
Fixed.